doc/configuration.rst

```diff
@@ -1443,6 +1443,8 @@ Arguments:
     (has precedence over `NetworkService`'s password)
   - stderr_merge (bool, default=False): set to True to make `run()` return stderr merged with
       stdout, and an empty list as second element.
+  - explicit_sftp_mode (bool, default=False): if set to True, `put()` and `get()` will
+    explicitly use the SFTP protocol for file transfers instead of the default SCP protocol
```

Comment:
```suggestion
    explicitly use the SFTP protocol for file transfers instead of the default protocol
```
or:
```suggestion
    explicitly use the SFTP protocol for file transfers instead of scp's default protocol
```

I did not notice this earlier: "default SCP protocol" is still misleading. The default protocol might be SFTP, depending on the version.

Reply:
I guess we could be even more specific and write something like "... explicitly use the SFTP protocol for file transfers instead of scp's default protocol if 8.6 >= OpenSSH < 9.0". I am not sure if this will cause confusion though. Maybe it is better to keep the description simple like your suggestion. What do you think?

Reply:
It's okay for me either way.